lib/unitarize_links_quda.cu

```diff
@@ -121,8 +121,21 @@ namespace quda {
     void apply(const qudaStream_t &stream)
     {
       TuneParam tp = tuneLaunch(*this, getTuning(), getVerbosity());
-      launch<Unitarize>(tp, stream,
-                        UnitarizeArg<Float, nColor, recon>(out, in, fails, max_iter, unitarize_eps, max_error, reunit_allow_svd, reunit_svd_only, svd_rel_error, svd_abs_error));
+      if (in.StaggeredPhase() == QUDA_STAGGERED_PHASE_MILC) {
```

Comment:
This needs an instantiation for `in.StaggeredPhase() == QUDA_STAGGERED_PHASE_NO`